src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elasticsearch-monitoring-integration.mdx

```diff
@@ -246,7 +246,7 @@ The Elasticsearch integration collects these metrics for clusters:
       </td>
 
       <td>
-        The remaining CPU credits available for data nodes in the cluster. A CPU credit provides the performance of a full CPU core for one minute. This metrics is available only for the t2.micro.elasticsearch, t2.small.elasticsearch, and t2.medium.elasticsearch instance types.
+        The remaining CPU credits available for data nodes in the cluster. A CPU credit provides the performance of a full CPU core for one minute. This metrics is available only for the t2.micro.search, t2.small.search, and t2.medium.search instance types.
```

Comment:
I think that the instances could go better between ``? And the word `metrics` should go in singular or should go in plural? `These metrics are available...`.

_This metric is available only for the `t2.micro.search`, `t2.small.search`, and `t2.medium.search` instance types._